feats/redis.py

```diff
@@ -103,7 +112,8 @@ class RedisClient:
     Initializing this class initializes the redis connection client but does
     _not_ test the connection to redis server.
     """
-    def __init__(self, host='localhost', port=6379, db=0, **options):
+    def __init__(self, host='localhost', port=6379, db=0, stream_prefix=None, **options):
```

Comment:
```suggestion
    def __init__(self, host='localhost', port=6379, db=0, key_prefix=None, **options):
```
We need any keys we create, regardless of if they are a stream or not, to be separate from other users.